app/src/main/java/com/github/bgabriel998/softwaredevproject/camera/CameraUiView.java

```diff
@@ -342,7 +342,7 @@ else if(POIPoints != null && !POIPoints.isEmpty()){
     private void drawPOIs(int actualDegree){
         //Go through all POIPoints
         for(POIPoint poiPoint : POIPoints){
-            if((int)poiPoint.getHorizontalBearing() == actualDegree){
+            if((int)poiPoint.getHorizontalBearing() == (actualDegree + 360) % 360){
```

Comment:
```suggestion
            if((int)poiPoint.getHorizontalBearing() == actualDegree % 360){
```

Reply:
I just checked online and in code and actually the modulo operator % in java returns negative values